lib/elixir/lib/module.ex

```diff
@@ -1752,15 +1806,18 @@ defmodule Module do
       {line, doc} when is_integer(line) and (is_binary(doc) or is_boolean(doc) or is_nil(doc)) ->
         value
 
+      {line, [{key, _} | _]} when is_integer(line) and is_atom(key) ->
+        value
+
       {line, doc} when is_integer(line) ->
         raise ArgumentError,
               "@#{key} is a built-in module attribute for documentation. It should be " <>
-                "a binary, a boolean, or nil, got: #{inspect(doc)}"
+                "a binary, boolean, keyword list, or nil, got: #{inspect(doc)}"
```

Comment:
Not a native English speaker here, but “It should be a binary, a bookean, a keyword list or nil” sounds better to me :)

(same applies to the text below)

Reply:
I don't know, also not a native speaker, and for me including an indefinite article before each item sounds superfluous. I tried now to look up, but couldn't find a generic rule for this. This is the closest I've come: https://english.stackexchange.com/questions/253041/should-an-article-be-used-when-enumerating-items

My favorite bit from that page:

> shows that the rules governing article usage are highly complex